source/manual/local-frontend-development.html.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Local frontend development
+parent: "/manual.html"
+layout: manual_layout
+section: Frontend
+type: learn
+owner_slack: "#govuk-2ndline"
+last_reviewed_on: 2019-07-08
+review_in: 6 months
+---
+
+Make the following changes to your local setup:
+
+In [govuk_app_config], add your local domain to the allowed CORS domains:
+
+```
+  GOVUK_DOMAINS = [
+    '*.publishing.service.gov.uk',
+    # "*.#{ENV['GOVUK_APP_DOMAIN_EXTERNAL'] || ENV['GOVUK_APP_DOMAIN'] || 'dev.gov.uk'}"
```

Comment:
I think it would be clearer to remove this line, as it's not immediately obvious that it's being commented out here?

Reply:
Also, we should probably make clear this is only a temporary workaround. This should be fixed within `govuk_app_config` really.

Reply:
Fixed in https://github.com/alphagov/govuk_app_config/pull/100